Dockerfile

```diff
@@ -6,5 +6,11 @@ RUN pip install -r /tmp/requirements.txt
 WORKDIR /opt/iscsc.fr-notify-bot
 COPY . .
 
+# root group already exists, but we need to create the blog_root user
+# Documentation for RUN useradd : https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user
+# Documentation for USER : https://docs.docker.com/engine/reference/builder/#user
+RUN useradd --no-log-init -r -g root blog_root
```

Comment:
also in scripts like this we prefer to use long flags like `--groups` instead of `-g` far easier to be understand later!
The short options are preferred to be used in interactive shells not scripts :wink:

Reply:
tks!